src/site.rs

```diff
@@ -178,6 +182,30 @@ fn author_map_to_scores(map: &AuthorMap) -> Vec<Entry> {
     scores
 }
 
+/// Deduplicate scores based on the assumption that an e-mail uniquely identifies a given
+/// person. If there are multiple entries with the same email, their commit counts will be
+/// merged into a single entry, with the canonical name being chosen based on the entry with
+/// the most commits.
+fn deduplicate_scores(entries: Vec<Entry>) -> Vec<Entry> {
+    let mut entry_map: HashMap<String, Vec<Entry>> = HashMap::with_capacity(entries.len());
+    for entry in entries {
+        entry_map.entry(entry.email.clone()).or_default().push(entry);
+    }
+
+    entry_map.into_values().map(|mut entry| {
+        // If there are multiple entries with the same maximum commit count, ensure that
+        // the ordering is stable, by sorting based on the whole entry.
+        entry.sort();
+        let canonical_entry = entry.iter().max_by_key(|entry| entry.commits).unwrap();
```

Comment:
can you also add the author as a secondary key? for cases where multiple people have the same commit

Reply:
oh nvm